Remotion/Web/Core/Res/Themes/NovaGray/HTML/TreeView.css

```diff
@@ -189,6 +183,15 @@ span.treeViewNode ul
   right: auto;
 }
 
+span.treeViewNodeBadge
+{
+  vertical-align: -6px;
```

Comment:
about `-0.47em` would be match the `6px`. With `-0.4em` there is a 1px gap

Reply:
hmm...Okay, so I think this may be from the following snippet in line 181
```
span.treeViewNodeHead span,
span.treeViewNodeHeadSelected span
{
  vertical-align: -1px;
}
```
This -1px could be on top of the -0.4em. could you please verify this and then we will make the vertical-align of the badge as `calc(-0.4em - 1px)`